amplify-tools/gradle.properties

```diff
@@ -0,0 +1,36 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx1536m
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier=true
+
+GROUP=com.amplifyframework
+VERSION_NAME=1.0.0
+AWS_CORE_SDK_VERSION=2.16.3
+
+POM_URL=https://github.com/aws-amplify/amplify-cli/
+POM_SCM_URL=https://github.com/aws-amplify/amplify-cli/
+POM_SCM_CONNECTION=scm:git:git://github.com/aws-amplify/amplify-cli.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/aws-amplify/amplify-cli.git
```

Comment:
Update these so that they are correct and refer to the current state of the project in `amplify-android`.